src/AutoRest.CSharp/DataPlane/Output/DataPlaneClient.cs

```diff
@@ -29,6 +29,7 @@ public DataPlaneClient(OperationGroup operationGroup, BuildContext<DataPlaneOutp
         {
             _operationGroup = operationGroup;
             _context = context;
+            DefaultAccessibility = "public";
```

Comment:
public is the default

Reply:
obsolete given comment below

Reply:
I guess each client has to define accessibility now.